client/extensions/zoninator/components/settings/zone/post-card.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import SectionHeader from 'components/section-header';
+
+class PostCard extends Component {
+
+	handleRemove = () => {
+		this.props.remove( this.props.cardIdx );
+	}
+
+	handleMouseDown = ( event ) => {
+		event.stopPropagation();
+		event.preventDefault();
+	}
+
+	render() {
+		const {
+			translate,
+			post: {
+				ID,
+				URL,
+				site_ID,
+				title,
+				slug,
+			}
+		} = this.props;
+
+		const postCardClass = 'zoninator__zone__list-item';
+
+		return (
+			<SectionHeader key={ slug } label={ title } className={ postCardClass }>
+				<Button
+					compact
+					onMouseDown={ this.handleMouseDown }
+					href={ URL }
+					target="_blank"
+					rel="noopener noreferrer">
+					{ translate( 'View' ) }
+				</Button>
+				<Button
+					compact
+					onMouseDown={ this.handleMouseDown }
+					href={ `/post/${ site_ID }/${ ID }` }
```

Comment:
We even have a [`getEditorPath()` selector](https://github.com/Automattic/wp-calypso/blob/3f18a57e24e6cb31c24f786dffa79ba1fc9bd2f8/client/state/ui/editor/selectors.js#L72-L90) 😄